src/qatozil.c

```diff
@@ -1,6 +1,10 @@
 #include <stdbool.h>
 #include <stdint.h>
+#ifndef UNIT_TESTS
 #include <string.h>
+#else
+#include <bsd/string.h>
```

Comment:
Out of curiosity, what is the reason for this distinction?

Reply:
It is to include `strlcpy` which is not declared on the default `string.h` but only on `bsd` one.
That only for the unit tests, because for building the app, we are using another libc `string.h` which includes the definition of `strlcpy`.

Reply:
Thanks